error in function "get_community_events"

Hi,
I'm facing a problem when trying to run the demo, specifically when running the make command "run-actions". The error points to the function "get_community_events" in https://github.com/RasaHQ/rasa-demo/blob/master/demo/community_events.py:
`Traceback (most recent call last):
  File "/usr/local/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.6/site-packages/rasa_core_sdk/endpoint.py", line 143, in <module>
    main(cmdline_args)
  File "/usr/local/lib/python3.6/site-packages/rasa_core_sdk/endpoint.py", line 135, in main
    run(args.actions, args.port, args.cors)
  File "/usr/local/lib/python3.6/site-packages/rasa_core_sdk/endpoint.py", line 119, in run
    edp_app = endpoint_app(cors_origins=cors, action_package_name=actions)
  File "/usr/local/lib/python3.6/site-packages/rasa_core_sdk/endpoint.py", line 42, in endpoint_app
    executor.register_package(action_package_name)
  File "/usr/local/lib/python3.6/site-packages/rasa_core_sdk/executor.py", line 164, in register_package
    self.register_action(action)
  File "/usr/local/lib/python3.6/site-packages/rasa_core_sdk/executor.py", line 105, in register_action
    action = action()
  File "/app/demo/actions.py", line 564, in __init__
    self.events = self._get_events()
  File "/app/demo/actions.py", line 575, in _get_events
    self.events = get_community_events()
  File "/app/demo/community_events.py", line 84, in get_community_events
    events = soup.find("ul", attrs={"class": "bulleted"}).find_all("li")
AttributeError: 'NoneType' object has no attribute 'find_all'
Makefile:16: recipe for target 'run-actions' failed`

It seems like the html element on the community page (https://rasa.com/community/join/) that contains the community events has no longer the class "bulleted". Hence the events object becomes None.

--- a/demo/community_events.py
+++ b/demo/community_events.py
@@ -81,7 +81,7 @@ def get_community_events() -> List[CommunityEvent]:
 
         soup = BeautifulSoup(community_page, "html.parser")
 
-        events = soup.find("ul", attrs={"class": "bulleted"}).find_all("li")
+        events = soup.find_all("div", attrs={"class": "Grid__Column-sc-18qgnlv-1 dYmpGd"})[1].find("ul").find_all("li")
         events = [CommunityEvent.from_html(e) for e in events]
 
         now = datetime.date.today()

